tests/ops/api/v1/endpoints/test_system.py

```diff
@@ -33,7 +38,37 @@ def url_invalid_system() -> str:
     return V1_URL_PREFIX + f"/system/not-a-real-system/connection"
 
 
-class TestPatchSystemConnections(TestPatchConnections):
```

Comment:
nit: this struck me as a bit odd. after a bit of digging, i realize it makes sense - this test file/module is dedicated to tests on the system _connections_ endpoints, and so having these fixtures at the module level makes sense. but it made me wonder whether we should just tweak the file/module name to something like `test_system_connections.py`, so it's a bit clearer? the tests of the "base" system endpoints comes in `test_api.py`, so with as things are now, it's a bit confusing to track down which tests are where!

(i realize this is nothing you've changed in this PR, but just noticed as i was going through this! so maybe it makes sense to adjust here.)